src/shifter.c

```diff
@@ -188,11 +188,11 @@ int main(int argc, char **argv) {
 
         if (entry != NULL) {
             opts.args[0] = strdup(entry);
-            if (imageData.workdir != NULL) {
-                opts.workdir = strdup(imageData.workdir);
-            }
         }
     }
+    if (imageData.workdir != NULL && opts.workdir == NULL) {
```

Comment:
so this would mean that if the user doesn't explicitly override an image-specified workdir then the image-specified workdir will be used by default.  At present, shifter will attempt to use the CWD from the external environment.

I think this is the right logic, but I suppose we'll need to include warnings about the change in default behavior.